integrations/src/test/resources/org/jenkinsci/plugins/casc/ArtifactManagerS3.yml

```diff
@@ -0,0 +1,12 @@
+---
+unclassified:
```

Comment:
IIUC it is going to change with https://github.com/jenkinsci/artifact-manager-s3-plugin/pull/67